javascript/javascript3/week2/index.js

```diff
@@ -0,0 +1,95 @@
+// Movies exercise
+const badMovies = [];
+const badMovies2000 = [];
+
+fetch(
+        "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    )
+    .then((response) => response.json())
+    .then((data) => {
+        data.forEach((element) => {
```

Comment:
You can also use the "filter" to find the bad movies.

        const badMovies= data.filter((movie) => movie.rating <5);
        console.log(badMovies);
        const badMovies2000= badMovies.filter((movie) => movie.year > 2000);
        console.log(badMovies2000);

Reply:
Thank you so much for reviewing my homework!